css/styles.css

```diff
@@ -0,0 +1,123 @@
+.background-image {
```

Comment:
There should be no reason to make this a separate element, you should apply this background image to the `body` element